src/UI/Menu.elm

```diff
@@ -76,24 +93,32 @@ type alias InternalMenuItem msg =
             [ Menu.item Download
                 { icon = Icon.download
                 , title = "Download"
-                , color = Nothing
                 }
             ]
         |> Menu.renderElement renderConfig
 
 -}
-item :
-    msg
-    ->
-        { icon : Maybe (String -> Icon)
-        , title : String
-        , color : Maybe Color
-        }
-    -> MenuItem msg
-item onToggle { icon, title, color } =
-    MenuItem { onClick = onToggle, icon = icon, title = title, color = color }
+item : msg -> { icon : Maybe (String -> Icon), title : String } -> MenuItem msg
```

Comment:
1 - I don't really like this record here, the types are very different for any confusion.
2 - The icon is a  maybe? it seems different from the documentation. If it's a maybe, then let's move it to with-pattern too...

Reply:
1) I think it's better as a record, but it could be plain arguments as well

2) Never had to use it as a maybe, I was being overly careful with options here :/